homeassistant/components/blebox/const.py

```diff
@@ -28,6 +29,7 @@
     "gate": CoverDeviceClass.GATE,
     "relay": SwitchDeviceClass.SWITCH,
     "temperature": SensorDeviceClass.TEMPERATURE,
+    "button": ButtonDeviceClass.UPDATE,
```

Comment:
I think it is confusing to have a mix of Cover/Switch/Sensor/Button device classes.
I think this constant should exist instead within each platform (cover plaform contains only CoverDeviceClass, etc.)

Reply:
I've divided this dictionary and placed it's elements to contextually proper modules.